N34: miniquad (#1052)
rust-gamedev/rust-gamedev.github.io: content/news/034/index.md

## Engine Updates

### [miniquad]

[![diff](miniquad_pr.png)][miniquad-pr]

[miniquad] is a pure rust, cross-platform graphics library.

[The biggest PR in miniquad's history][miniquad-pr] landed this month, finishing
the effort of removing all the `sapp` legacy.

Changes includes:

> - no more `sapp-*` crates. Now its just one crate, the miniquad!
>   No mess with individual crates anymore, no ffi for
>   miniquad - sapp communication.
> - on linux miniquad do not depend/statically link with lib*-dev packages.  
>   Miniquad can choose between glx/egl, x11/wayland at runtime.
> - MacOS implementation do not depend on any Objective C code anymore.

Reviewer: ```suggestion
- No more `sapp-*` crates. Now it's just one crate, the miniquad!
  No mess with individual crates anymore, no ffi for
  miniquad - sapp communication.
- On Linux, miniquad does not depend/statically link with lib*-dev packages.  
  Miniquad can choose between glx/egl, x11/wayland at runtime.
- MacOS implementation does not depend on any Objective C code anymore.
```
